### DISCUSSION TITLE
Integration einer Währung/eines Shops

### DISCUSSION THREAD

**Discussion by wwwLuis** [SE-TINF22B2/G5-DuoGradus#9] [Anforderungen]
User sollen durch Aktivitäten und Achievements usw. eine Währung sammeln können, über die man bspw.: 
- eine kaputte Streak reparieren kann
- evtl. Styles für die App o.ä. in einem Shop kaufen kann

**Comment by benedictweis** [comment]
Sehr gute Idee, ähnlich bei Duolingo. Ich würde allerdings diese, nicht wie bei Duolingo, nicht "kaufbar" mache, sondern ausschließlich "erspielbar".

**Reply by wwwLuis** [reply]
alternativ können wir auch einfach mein paypal hinterlegen und das ganze mit echtgeld verknüpfen
